community-operators/jenkins-operator/0.3.31/jenkins-operator.v0.3.31.clusterserviceversion.yaml

```diff
@@ -211,6 +211,20 @@ spec:
                 name: jenkins-operator
                 resources: {}
               serviceAccountName: jenkins-operator
+      clusterPermissions:
+      - serviceAccountName: jenkins-operator
+        rules:
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - clusteroles
```

Comment:
This one does not seem correct to me:
the jenkins-operator does not need to create/update clusterroles nor clusterrolebinding.

But instead, it needs to be able to create rolebindings.

Reply:
updated